packages/frontend/src/wallet.d.ts

```diff
@@ -6,15 +6,15 @@ declare module Wallet {
         contractName?: string;
         balance: string;
         onChainFTMetadata: {
-            decimals: number;
-            icon: string;
-            name: string;
-            reference: string | null;
-            reference_hash: string | null;
-            spec: string;
-            symbol: string;
+            decimals?: number;
+            icon?: string;
+            name?: string;
+            reference?: string | null;
+            reference_hash?: string | null;
+            spec?: string;
+            symbol?: string;
```

Comment:
Based on NEP 0148, these are all mandatory fields when creating a new FT contract. It might be better to keep them as required

Reply:
The issue and why I did this is that a token is sent through as a prop throughout the app and only some of the properties are referenced at a time